src/app/+item-page/edit-item-page/item-relationships/edit-relationship-list/edit-relationship-list.component.html

```diff
@@ -1,15 +1,26 @@
-<h5>{{getRelationshipMessageKey() | async | translate}}</h5>
+<h5>
+    {{getRelationshipMessageKey() | async | translate}}
```

Comment:
One thing that is making me confused is having multiple "relations with the same name":

![image](https://user-images.githubusercontent.com/877488/88295377-40891180-ccf5-11ea-8721-4adc9d5ca136.png)

I know that this is related with my model, in this particular case, a Person in the point-of-view of a Publication, can be an Author or a Contributor. So, to me, the correct thing here is to differentiate those two situations. 
`Person as a Contributor of:` and `Person as an Author of:`

Reply:
where are the labels obtained from?

For a specific publication, these are my relations:
![image](https://user-images.githubusercontent.com/877488/88302876-446d6180-ccfe-11ea-902d-c5a44de96881.png)

This is my relationships type table: 
![image](https://user-images.githubusercontent.com/877488/88303816-5f8ca100-ccff-11ea-89dd-e31eee7a0225.png)


But translations are used to represent the relationship groups like:
`relationships.isContributorOf` or `relationships.isPublicationOf`


I don
The relation has two sides, like in my case: `isContributorOfPublication` and `isPublicationOfContributor`. 



and I think the wrong side

Reply:
Hi Paulo, this is a bit confusing indeed.
I've altered the getRelationshipMessageKey() method so it also takes the related entity type label into account: when it is different from the relationship label (e.g. OrgUnit <> isContributorOfPublication), it is appended to the message key, so a more specific label can be shown.